sopel-modules/rat-board.py

```diff
@@ -1820,34 +1854,97 @@ def cmd_nick(bot, trigger, case, newnick):
     save_case_later(bot, case, forceFull=True)
     bot.say('Set Nick to ' + str(newnick))
 
-@commands('quiet', 'lastsignal', 'last')
-@require_rat('Sorry, but you need to be a registered and drilled Rat to use this command.')
-def cmd_quiet(bot, trigger):
+
+def func_quiet(bot, trigger, platform=''):
     """
     Tells the time since the last Signal
     """
-    if bot.memory['ratbot']['lastsignal'] is None:
-        bot.say("Sadly, I don't remember when we had the last signal. Maybe it was 42 seconds ago?")
-        return
-    tdelta = datetime.datetime.utcnow() - bot.memory['ratbot']['lastsignal']
+    retVal = None
+    if trigger.is_privmsg: retVal = NOLIMIT
+    # returning NOLIMIT doenst trigger the rate-limit
+    # therefor PMs arent rate-limited
+    # any other returned value is ignored
+
+    retPlat = ''
+    if platform == 'xb' : retPlat = 'Xbox'
```

Comment:
`if : action` on same line
inconsistent with https://github.com/FuelRats/pipsqueak/pull/243/files#diff-7bc69ee12d552a3d373831306d1dc075R1874
Also, `elif` statements would be more effective here.

Reply:
It's consistent. Single action if is a one liner, multiple action if spreads across. And yes, elif would be more effective, but by how MUCH?